docs/source/development.rst

```diff
@@ -25,9 +25,17 @@ libraries. On Debian or Ubuntu, we can install these using
 
     $ sudo apt-get install python-dev zlib1g-dev git libxslt1-dev
 
-.. note::
-    TODO: Document this basic step for other platforms? We definitely want
-    to tell people how to do this with Brew or ports on a Mac.
+Alternatively, on Mac computers, install Anaconda Python
```

Comment:
The advantage is stuff like numpy and scipy are built in so you don't have to pip install them. They are a mess to install from pip, so this is a good thing. However, if you use an operating system with an actual software manager, installing numpy and scipy is easy so I've never bothered with it.